sdk/python/tests/dsl/component_tests.py

```diff
@@ -29,7 +29,7 @@ def _set_metadata(self, component_meta):
         self._metadata = component_meta
 
     @component
-    def componentA(a: {'Schema': {'file_type': 'csv'}}, b: Integer() = 12, c: GCSPath(path_type='file', file_type='tsv') = 'gs://hello/world') -> {'model': Integer()}:
+    def componentA(a: {'Schema': {'file_type': 'csv'}}, b: Integer() = 12, c: {'GCSPath': {'path_type': 'file', 'file_type':'tsv'}} = 'gs://hello/world') -> {'model': Integer()}:
```

Comment:
I thought that you are going to remove this properties for artifact.

Reply:
Yes, from all the user facing interfaces. 
I kept the unit tests to verify if the codebase works with the property lists.
I'm ok with removing them if it causes confusion, though.

Reply:
Maybe rename the properties to be not artifact related.

Reply:
OK, let me do that.

Reply:
Done.